tests/run-make/raw-dylib-cross-compilation/Makefile

```diff
@@ -1,8 +1,7 @@
 # Tests that raw-dylib cross compilation works correctly
 
 # only-gnu
```

Comment:
Probably silly, but was there a reason this `only-gnu` was left in place while the other (.rs) tests removed theirs?

Reply:
It was an oversight.